<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfferAcceptInfoDialog</class>
 <widget class="QDialog" name="OfferAcceptInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offer Accept Info</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="acceptinfo">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>501</width>
         <height>661</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Offer Accept Details</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="guidEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="linkGUIDLabel">
          <property name="text">
           <string>OfferLink ID:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="linkGUIDEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="certLabel">
          <property name="text">
           <string>Cert. ID:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="certEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="escrowLabel">
          <property name="text">
           <string>Escrow ID</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="escrowEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="btctxidLabel">
          <property name="text">
           <string>BTC TXID:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="btctxidEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>TXID:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="txidEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Offer Title:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="titleEdit"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="heightEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="timeEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Currency:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="currencyEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Price:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="priceEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="commissionLabel">
          <property name="text">
           <string>Commission:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="commissionEdit"/>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="discountLabel">
          <property name="text">
           <string>Discount:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="discountEdit"/>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Quantity:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="quantityEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLineEdit" name="totalEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Paid:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLineEdit" name="paidEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Average Rating:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLineEdit" name="ratingEdit"/>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Buyer:</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLineEdit" name="buyerEdit"/>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Shipping &amp; Contact Information:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QTextEdit" name="paymessageEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Seller:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLineEdit" name="sellerEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="buyerFeedback">
      <attribute name="title">
       <string>&amp;Buyer Feedback</string>
      </attribute>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>511</width>
         <height>671</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>509</width>
          <height>669</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="buyerFeedbackLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="sellerFeedback">
      <attribute name="title">
       <string>&amp;Seller Feedback</string>
      </attribute>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>511</width>
         <height>671</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>509</width>
          <height>669</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="sellerFeedbackLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syscoin.qrc"/>
 </resources>
 <connections/>
</ui>
